packages/cli/src/lib.rs

```diff
@@ -241,7 +241,7 @@ fn to_valid_pkg_name(project_name: &str) -> String {
         .replace('\\', "")
         .replace('/', "");
 
-    if let Some(ch) = ret.chars().next() {
+    for ch in ret.clone().chars() {
         if ch.is_ascii_digit() || ch == '-' {
             ret.remove(0);
         }
```

Comment:
we only need to remove the leading numbers; this however will remove the first character no matter what it is whenever a number is encountered in the iteration; instead, we want to find the index of the first character that is not a number nor a `-` and remove until it.

Reply:
Thanks for pointing out the problem, I'll fix it.

Reply:
Done.

Reply:
In addition, I added another commit, which added more tests.